src/main/java/org/codehaus/mojo/flatten/FlattenMode.java

```diff
@@ -72,7 +72,10 @@
     clean,
 
     /** Removes all {@link FlattenDescriptor optional POM elements} and dependencies. */
-    fatjar;
+    fatjar,
+    
+    /** Only flattens the version field in pom. Keeps everything else. */
+    versionOnly;

```

Comment:
as already commented: maybe `onlyResolveVersion` or `onlyExpandVersion` would be more self-explanatory.

Reply:
I will change that.